blockchain/chain/src/store/chain_store.rs

```diff
@@ -306,6 +306,14 @@ where
         })?)
     }
 
+    pub fn unmark_block_as_validated(&self, cid: &Cid) -> Result<(), Error> {
```

Comment:
Conceptually, it's a mistake that the set of validated blocks is saved to a file. But fixing it depends on issue #2771. Once that issue has been closed, this function will be infallible.